lib/honeybadger/trace.rb

```diff
@@ -45,8 +46,10 @@ def add_query(event)
       end
     end
 
-    def complete(event)
+    def complete(event, config)
       @meta = clean_event(event).to_h
+      @meta[:params] = Util::Sanitizer.new(filters: config.params_filters).sanitize(event.payload[:params]) if event.payload[:params]
```

Comment:
Is `event.payload[:params]` always referring to controller params? The current rack request is available via `config#request`, so we could use that to also add session/cookies/cgi_data. Basically I was thinking of just sending the same `request` hash that we do for notice payloads.